hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -590,11 +611,11 @@ private static void deleteKeysFromDelKeyTableInSnapshotScope(
   }
 
   // Get OmSnapshot if the keyName has ".snapshot" key indicator
+  @SuppressWarnings("unchecked")
   public ReferenceCounted<IOmMetadataReader, SnapshotCache> checkForSnapshot(
```

Comment:
nit: should this be called `getActiveFsOrSnapshotMetadata`? Or something along the lines.

`checkForSnapshot` feels like would return boolean response.